app/event-gateway/entities/virtual-cluster.md

```diff
@@ -0,0 +1,362 @@
+---
+title: "Virtual clusters"
+content_type: reference
+layout: gateway_entity
+
+description: |
+    Virtual clusters are {{site.event_gateway_short}} entities that expose a modified view of the backend cluster to clients.
+related_resources:
+  - text: "{{site.event_gateway}} Policy Hub"
+    url: /event-gateway/policies/
+  - text: "Policies"
+    url: /event-gateway/entities/policies/
+  - text: "Backend Clusters"
+    url: /event-gateway/entities/backend-clusters/
+  - text: "Listeners"
+    url: /event-gateway/entities/listeners/
+
+tools:
+    - konnect-api
+    - terraform
+tags:
+  - policy
+
+works_on:
+  - konnect
+
+schema:
+    api: event-gateway/knep
+    path: /schemas/VirtualCluster
+
+api_specs:
+    - event-gateway/knep
+
+products:
+    - event-gateway
+---
+
+## What is a virtual cluster?
+
+Virtual clusters are the primary way clients interact with the {{site.event_gateway_short}} proxy. 
+They allow you to isolate clients from each other when connecting to the same [backend cluster](/event-gateway/entities/backend-clusters/), 
+and provide each client with modified view while still appearing as a standard Kafka cluster.
+
+Here's how it works:
+1. The Kafka client produces an event.
+1. A listener forwards it to the correct virtual cluster.
+1. The virtual cluster applies policies and proxies the modified event data to the backend cluster.
+1. The backend cluster, representing a Kafka cluster, receives data and sends a response.
+
+{% mermaid %}
+flowchart LR
+    A[Kafka client] --> B[Listener
+    + listener policies]
+    B --> C[Virtual cluster
+    + consume, produce, and cluster policies]
+    C --> D[Backend 
+    cluster]
+
+    style C stroke:#86e2cc
+{% endmermaid %}
```

Comment:
Can we do another arrow from the backend cluster so that it shows it sending a response? For now it looks like it just sends the event data to the backend cluster and it stops.